game/views.py

```
@@ -1770,20 +1770,27 @@ def stats_view(request):
-    ).order_by(
-        "-puzzles_solved",
-        "-best_streak"
-    )
+    try:
+        leaderboard = PuzzleStats.objects.select_related(
+            "user"
+        ).order_by(
+            "-puzzles_solved",
+            "-best_streak"
+        )
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)